packages/spruce-node/package.json

```diff
@@ -17,10 +17,14 @@
 	},
 	"scripts": {
 		"lint": "eslint '**/**/*{.js}'",
-		"#test": "jest"
+		"#test": "jest",
+		"syncApiSchema:local": "API_HOST=https://local-api.sprucebot.com/graphql node ./mock/fetchApiSchema.js"
```

Comment:
Yeah, the `spruce-node` tests are pretty outdated and weren't really testing much of anything...and they'd often cause phantom failures. It might be worth revisiting tests now against the mock server w/ this PR